plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,81 +3,86 @@ p.
+bq. *Please note:* In automata theory an automaton reacts on inputs and produces outputs. There, the terms input and output are usually used for text inputs/outputs. Modern state machines use an extended definition of inputs and outputs. Inputs can be events like a button click or a time trigger while outputs can be all kinds of expressions or operation calls.
 
-h2(#overview_moore). States and transitions – The ferry as a Moore machine
+In the following, we will extend the simple switch example to explain the differences between Mealy and Moore machines as well as Harel statecharts and UML state machines.
 
-In automata theory, there are two basic types of finite state machines (FSM). One of those is called "_Moore_ machine":https://en.wikipedia.org/wiki/Moore_machine, named after its inventor Edward Moore. Moore machines consist of states and transitions. States are able to produce outputs, and the output is determined solely by the current state, not by any input. Modeled as a Moore machine the ferry example looks like the following:
 
-!(large)images/ferry_moore.png(Ferry example as a Moore machine, modeled with YAKINDU Statechart Tools)!
+*Moore machines*
 
-p=. Ferry example as a Moore machine, modeled with YAKINDU Statechart Tools
+In automata theory, there are two basic types of finite-state machines (FSM). One of those is called "_Moore_ machine":https://en.wikipedia.org/wiki/Moore_machine, named after its inventor Edward Moore. Moore machines consist of states and transitions. *States are able to produce outputs*, and the output is determined solely by the current state, not by any input.
 
-The state diagram has four states: *NorthernPort*, *SouthernPort*, *Northwards* and *Southwards*. Each of these states produces an output which is an action of the ferry. The ferry can either stop when one of the two ports is reached (_ferry.stop_) or move from one port to another (_ferry.moveSouthwards_ and _ferry.moveNorthwards_). In Moore machines the output *only* relies on the current state. This means that the output will not change unless the state also changes. This happens when a transition is taken. In our example, we have two kinds of inputs that may trigger a state transition: The ship’s mate who can decide where to sail (e.g. _mate.sailSouthwards_) and the quay which can detect an arriving ferry (e.g. _quay.reachedSouthernBank_).
+We extend the switch example above into a light switch with different brightness levels. The light switch has two buttons, an _ON_ button and an _OFF_ button. Pressing the _ON_ button switches the light on and toggles through the different brightness levels. The output of the state machine is the brightness value. As in Moore machines only states produce outputs, we need one dedicated state per brightness level:
```

Comment:
"Pressing the _ON_ button switches the light on and toggles through the different brightness levels"

This is not 100% correct. The _ON_ button raises the *ON_pressed* event. It does not turn on or off the light - then we would need no state machine, cause it is simply wired